File: CoreLib_Common/Model/Job.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CoreLib_Common.Model
{
    public class Job
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public string Title { get; set; } = null!;
        public int Salary { get; set; }

        public virtual ICollection<Animal>? Animals { get; set; }
        public virtual ICollection<JobDrawback>? JobDrawbacks { get; set; }

        public override string ToString()
        {
            return $"Job : Id = {Id} Title = {Title}";
        }
    }
}

File: VBLib_Common/Model/Job.vb
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Model
    Public Class Job
        <Key>
        <DatabaseGenerated(DatabaseGeneratedOption.Identity)>
        Public Property Id As Integer
        Public Property Title As String
        Public Property Salary As Integer
        Public Overridable Property Animals As ICollection(Of Animal)
        Public Overridable Property JobDrawbacks As ICollection(Of JobDrawback)

        Public Overrides Function ToString() As String
            Return $"Job : Id = {Id} Title = {Title}"
        End Function
    End Class
End Namespace
